app/src/main/java/org/fossasia/susi/ai/rest/responses/others/RelatedTopics.kt

```diff
@@ -0,0 +1,29 @@
+package org.fossasia.susi.ai.rest.responses.others
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+/**
+ * <h1>POJO class to parse related topics object in retrofit response from websearch client.</h1>
+
+ * Created by mayank on 09-12-2016.
+ */
+
+class RelatedTopics
+
+(@SerializedName("FirstURL")
+ @Expose
+ var url: String?,
+
+ @SerializedName("Text")
+ @Expose
+ var text: String?,
+
+ @SerializedName("Icon")
+ @Expose
+ var icon: WebIcon?) {
+
+    @SerializedName("Result")
+    @Expose
+    var result: String? = null
+}
```

Comment:
As far as I know, there is no need of explicitly defining a no argument constructor. The compiler does that for you. 
https://stackoverflow.com/questions/3641114/do-i-really-need-to-define-default-constructor-in-java
According to 'documentation' https://docs.oracle.com/javase/tutorial/java/javaOO/constructors.html 
>You don't have to provide any constructors for your class, but you must be careful when doing this. The compiler automatically provides a no-argument, default constructor for any class without constructors. 

Correct me if I am wrong but all these POJO classes are working without defining a constructor explicitly. https://github.com/fossasia/susi_android/tree/development/app/src/main/java/org/fossasia/susi/ai/rest/responses/susi 

Plus I am just asking, you haven't explicitly used () these brackets anywhere else like 
https://github.com/fossasia/susi_android/pull/838/files#diff-3d6fc652d7495b5553ae8e938dd29431
https://github.com/fossasia/susi_android/pull/838/files#diff-6829e258eaaa5121e5ff77cc43e26b8d
https://github.com/fossasia/susi_android/pull/838/files#diff-ddd769f6bd55d9cc0b3515c5dc4319ce
https://github.com/fossasia/susi_android/pull/838/files#diff-30f6b7e474a3eea82dbd03ddee3fb2b7
https://github.com/fossasia/susi_android/pull/838/files#diff-2ae65d763b750ec12c30376a97ccbddd
https://github.com/fossasia/susi_android/pull/838/files#diff-e9a443de7517d5d06c21bc5b85588fdd
https://github.com/fossasia/susi_android/pull/838/files#diff-d2eb5bc4d9257b98916861793a168a62
https://github.com/fossasia/susi_android/pull/838/files#diff-f34134b4b2b04e01a94ba8dfe11ad340
etc. Why is this class special?

According to me, we don't need to specifically use () brackets here. Then again, correct me if I am wrong.